lib/ex_unit/lib/ex_unit/diff.ex

```diff
@@ -66,6 +71,18 @@ defmodule ExUnit.Diff do
 
   def format(_left, _right, _formatter), do: nil
 
+  defp proceed_string_difference?(left, right) do
+    sum_length = String.length(left) + String.length(right)
```

Comment:
I wonder if `String.length` makes sense here. Wouldn't `binary_size` be enough? I'd say it's a good enough estimate of the length of the string and should be significantly faster especially for large strings - and that's what this PR is optimizing for.